attestator.py

```diff
@@ -46,6 +46,9 @@
 
 
 def generate_pdf(user_data):
+    for key in user_data:
+        assert (user_data[key] != ''), 'Erreur: le champs '+ key + ' est vide.'
+
     c = canvas.Canvas("attestation.pdf")
```

Comment:
```suggestion
def generate_pdf(user_data, output_file="attestation.pdf"):
    for key in user_data:
        assert (user_data[key] != ''), 'Erreur: le champs '+ key + ' est vide.'

    c = canvas.Canvas(output_file)
```

et comme ça dans ta lib, tu dois pouvoir faire un truc du genre:
```python
attestation = io.BytesIO()
generate_pdf(user_data, attestation)
```

Reply:
Fait. Mais malheureusement iframe me renvoie des 404 tout le temps. J'ai enlevé la visualisation du pdf.

Reply:
Ah mince. Bah n'hésite pas à refaire une PR si tu arrives à corriger le soucis. En attendant, on va déjà merge celle-ci. :)